### DISCUSSION TITLE
Segment Data

### DISCUSSION THREAD

**Discussion by Bernhard3012** [OpenBB-finance/OpenBB#4898] [Q&A]
Hi there, 
I unfortunately do not have any coding experience or knowledge, but I am trying to learn. I am looking for a way to get the segment data from the 10-Q and 10-K filings. Does anyone have any idea how to get it from the SEC filings?

Thanks and best regards
Bernhard

**Comment by markoblogo** [comment]
Here is a practical way to start:
- Use the SEC/filings endpoint (or the OpenBB CompanyFilings data model) to get the URLs of the company’s 10‑Q and 10‑K reports from EDGAR.
- Download the filing (HTML or text) from the URL.
- Search inside the document for the segment note (often called “Note on Segment Information”, “Segment Reporting” or similar) and extract the tables manually or with a simple script, depending on your comfort level with tools.
- If you are just getting started with coding, you can first do this manually in the browser/PDF viewer, and only later automate it once you are comfortable with Python and OpenBB.
